CMakeLists.txt

```diff
@@ -272,17 +274,24 @@ if(GENERATE_DOC)
     # Copy the aliases to the build directory
     file(COPY ${SOURCE_DIR}/docs/aliases DESTINATION ${BUILD_DIR}/docs)
 
+    add_custom_target(ldoc ALL
+        COMMAND ${LDOC_EXECUTABLE} .
+        WORKING_DIRECTORY ${AWE_DOC_DIR}
+        DEPENDS ${AWE_LUA_FILES} ${AWE_MD_FILES} ${BUILD_DIR}/docs/06-appearance.md
+        ${BUILD_DIR}/docs/05-awesomerc.md
+    )
```

Comment:
I think we should use `.../docs/*.md.lua` (with the `.lua` extension removed) in the DEPENDS here.